drakrun/drakrun/main.py

```diff
@@ -248,6 +248,12 @@ def process(self):
         magic_output = magic.from_buffer(sample.content)
         self.log.info("running sample sha256: {}".format(sha256sum))
 
+        timeout = self.current_task.payload.get('timeout') or 60 * 10
+        hard_time_limit = 60 * 20
```

Comment:
I wonder if this could be some global setting which affects both drakcore and all drakrun instances and where it would be appropriate to keep such settings. Maybe in Redis?

(Not needed to implement it within this PR, just thinking)

Reply:
I guess that redis is probably the best place to store it? :thinking:

Reply:
On the other hand, maybe just handle it in drakcore somehow? Like an API endpoint for global config?

Reply:
I think about the section `[global]` in drakcore config.ini which would be uploaded into Redis by our wrapped drak-system service each time it starts. Then, both drakcore and drakrun instances would have access to this information. Other idea is to have some global config uploaded into MinIO and downloaded by drakrun on start/restart.